controllers/genreController.js

```diff
@@ -46,27 +47,27 @@ exports.genre_create_get = function(req, res, next) {
 };
 
 // Handle Genre create on POST
-exports.genre_create_post = function(req, res, next) {
-
-    //Check that the name field is not empty
-    req.checkBody('name', 'Genre name required').notEmpty();
-
-    //Trim and escape the name field.
-    req.sanitize('name').escape();
-    req.sanitize('name').trim();
-
-    //Run the validators
-    var errors = req.validationErrors();
+exports.genre_create_post =  [
+    // validate and sanitize request
+   
+    //Check that the name field is not empty also trim and escape the name field.
+      body('name', 'Genre name required').isLength({ min: 1 }).trim().escape(),
```

Comment:
Did you test the validation? Specifically, set the min above to 5 and try entering something that requires trimming or escaping. When I tried this last Friday this did not work. The reason was something to with sanitising methods in the validation step only applying during validation. To propagate them you were supposed to use the matcheddata or sanitize. 
Now I may have had other problems with my code affecting this, but best we're sure!

Also, do we need to try/catch some of this?

Reply:
Thank you for your feedback.
Yep made a mistake shouldn't have combined validation and sanitization. I did ran couple of tests for validation thought jade is modifying the response accordingly on the rendered page. Fixed now. 

I guess we don't need try/catch block as none of the validation/sanitize method is throwing error.